### ISSUE TITLE
Our public IP address is listed in a Spamhaus blacklist

### ISSUE THREAD

**Issue by amcewen** [DoESLiverpool/somebody-should#850]
https://www.abuseat.org/lookup.cgi?ip=185.135.104.23

(Noticed because my email provider is refusing email from me due to the network being blacklisted).

From that blacklist URL it seems that there's a Windows machine somewhere on the network which is infected with the Conficker worm.  Seems it was a machine that was most recently on the network yesterday.

I've run a [scanning tool](http://net.cs.uni-bonn.de/wg/cs/applications/containing-conficker/) to look for infected machines (I had to change `impacket.dcerpc` to `impacket.dcerpc.v5` at the top of `scanner21.py` to get it to work) and that doesn't report any hosts on 10.0.29.x, 10.0.30.x or 10.0.11.x, which makes it look more like it's someone's own laptop that's infected...

@ajlennon is running a different scan tool too, so we'll see if he finds anything.

**Comment by amcewen** [comment]
Having checked on the Tapestry WiFi, that routes me out via the same IP address, so it might not be a machine on our network. @mattwilsondotuk, do you have any way to spot which host is trying to connect out to `104.244.14.252` to narrow down which machine is infected, alongside our local checks...?

**Comment by skos-ninja** [comment]
I have firewalled the IP that it was detected connecting to and enabled logging so if it tries connecting again I can see it

**Comment by ajlennon** [comment]
I have some Nessus scans running. Will upload when done

**Comment by ajlennon** [comment]
[Contrained_Backdoor_Scan_h76cbh.pdf](https://github.com/DoESLiverpool/somebody-should/files/2197936/Contrained_Backdoor_Scan_h76cbh.pdf)

**Comment by amcewen** [comment]
The [abuse alert page](https://www.abuseat.org/lookup.cgi?ip=185.135.104.23) has continued reporting issues each day (generally at 19:45-19:55) and @skos-ninja hasn't seen any alerts on the firewall (apart from the one from my laptop just now when I tried connecting out to test it), so it looks like our network is in the clear.

I told @mattwilsondotuk about it when I bumped into him the other evening, so he's aware of it.  I'm going to leave this open until we get our static IP (or the reported problem gets fixed) - we need to bug Jason about clearing the IX Liverpool server closet to get the static IP moved on.
